libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/BaseTest.java

```diff
@@ -44,6 +44,10 @@ protected RouteProgress buildDefaultTestRouteProgress() throws Exception {
     return routeProgressBuilder.buildDefaultTestRouteProgress(testRoute);
   }
 
+  protected RouteProgress buildDefaultTestRouteProgress(DirectionsRoute testRoute) throws Exception {
+    return routeProgressBuilder.buildDefaultTestRouteProgress(testRoute);
```

Comment:
I don't like `BaseTest` being the place to put all of our builders for test objects, can't the test classes call the TestBuilders directly? Also, I think we need to decide between `Test..Builder` and `...Faker` for naming for consistency because they seem to be doing the same thing